src/api/types.ts

```diff
@@ -0,0 +1,85 @@
+export interface Exercise {
+    [key: string]: string | string[] | undefined | '' | 'kg' | 'lb',
+    name: string,
+    img: string,
+    muscles?: string[],
+    sets?: string,
+    reps?: string,
+    weight?: string,
+    unit?: "" | "kg" | "lb",
+    rest?: string
+}
+
+export interface WholeExercise {
+    created_at: string,
+    id: number,
+    exercise: Exercise
+}
+
+export interface ExerciseResponse {
+    count: null,
```

Comment:
What is `count`?

Reply:
No idea, this is a supabase response, not my object.

I explained it, like, twice in my previous comments. Do you want me to repeat the same thing for the third time?

Reply:
But it comes form the backend you use, so I think you should know. IMHO that's a number and it can't be null

Reply:
So, **as far as I know**, the `count` is always `null` (take a look at screens below).

![error](https://user-images.githubusercontent.com/113309294/229305651-1569caa6-e6b6-4c9b-802e-20bff21c7b0d.PNG)
![ok](https://user-images.githubusercontent.com/113309294/229305652-b888eaa8-6c61-4581-a650-9b4978d513a2.PNG)

The only thing I don't know is if those two are the only possible structures of the response object.